Gini-iOS-SDK.xcodeproj/project.pbxproj

```diff
@@ -646,6 +646,7 @@
 						};
 					};
 					23D53C031934BC85001A957E = {
+						ProvisioningStyle = Automatic;
```

Comment:
Again, you have changes in the project file that are unrelated to the feature.

Reply:
I had to add it because for the tests weren't running. 
I want to migrate the whole project to a new structure, since it was set up with CocoaPods < 1.0 and It is less clear in my opinion (and more prone to errors).